docs/stellar-core_example.cfg

```diff
@@ -29,7 +29,7 @@ HISTOGRAM_WINDOW_SIZE=30
 # This will get written to a lot and will grow as the size of the ledger grows.
 BUCKET_DIR_PATH="buckets"
 
-
```

Comment:
You should add a comment here explaining this config. Also, I think it would be helpful to move this down over by the History section on line 686.

Reply:
Good shout. But maybe it should live in the catchup config section? (~331)